zingolib/src/lib.rs

```diff
@@ -21,6 +21,48 @@ use std::{
 use tokio::runtime::Runtime;
 use zingoconfig::{ChainType, ZingoConfig};
 
+#[cfg(feature = "async_client")]
```

Comment:
Why do we need to feature-gate this?

Reply:
This code isn't currently used by any consumer, _except_ the tests, I thought it made sense to gate it..   at least until when/if more consumers use it.